app/views/supervisor/dashboard/index.html.slim

```diff
@@ -0,0 +1,16 @@
+h1 = "Hi #{current_user.name}!"
+h2 This is your brandnew Supervisor Dashboard - with love from Team Cheesy
+p alpha version
+
+hr
+
+div.clearfix
+  - for t in supervised_teams.each do
```

Comment:
Euh, not sure what the thought process was here, but this should be `supervised_teams.each do |t|` instead.